src/type-layout.md

```diff
@@ -53,9 +53,25 @@ target platform. For example, on a 32 bit target, this is 4 bytes and on a 64
 bit target, this is 8 bytes.
 
 Most primitives are generally aligned to their size, although this is
-platform-specific behavior. In particular, on x86 u64 and f64 are only
+platform-specific behavior. In particular, on x86, `u64` and `f64` are only
```

Comment:
Hm, this phrasing struck me as potentially confusing, since "x86" is often used as a shorthand to refer to "x86-64". (I did a brief double take, at least.) Per Wikipedia's [chronology](https://en.wikipedia.org/wiki/X86#Chronology), "IA-32" might be a more appropriate term: 
```suggestion
platform-specific behavior. In particular, on IA-32, `u64` and `f64` are only
```

...but I think you're referring to [`target_arch`](https://doc.rust-lang.org/reference/conditional-compilation.html#target_arch) value "x86" (which is distinct from `x86_64`). Rust [inherits this shorthand from LLVM](https://llvm.org/doxygen/Triple_8h_source.html#l00085): 
```text
    x86,            // X86: i[3-9]86
```
...so you could instead disambiguate this by writing:
```suggestion
platform-specific behavior. In particular, on `target_arch = x86` (which has a word size of 32 bits), `u64` and `f64` are only
```